internal/config/config_print_test.go

```diff
@@ -0,0 +1,24 @@
+package config
+
+import (
+	"io"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"numerous.com/cli/internal/test"
+)
+
+func TestPrint(t *testing.T) {
+	stdoutR, stdoutW := test.PatchStdout(t)
+
+	c := Config{OrganizationSlug: "test-organization-slug"}
+	c.Print()
```

Comment:
couldn't you use the helper `RunWithPatchedStdout` here, to simplify the test?

Reply:
Good point! I refactored a bit, and used this instead.